core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml

```diff
@@ -8,3 +8,66 @@
     mode: u=rw,go=
   when: specification.alert_rules.common | bool
   notify: Reload Prometheus
+
+- name: Copy Prometheus alerting rules
+  copy:
+    src: "prometheus/rules/prometheus.rules"
```

Comment:
choose whether with or without quotation marks - i think we don't need "" in this case